project/plugins.sbt

```diff
@@ -1,5 +1,6 @@
 logLevel := Level.Warn
 
-addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.1")
-addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.28")
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.32")
 addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "0.6.1")
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.5.0")
```

Comment:
wouldn't that cause a conflict on project import for those having `sbt-updates` as a global sbt plugin at `~/.sbt/1.0/plugins/plugins.sbt` in case of the versions mismatch?

Reply:
I think it will be overridden by the project settings.

Reply:
I don't think we need this in the project itself.
We can setup Scala Steward if not done already.

Reply:
>I don't think we need this in the project itself.
I agree, it wasn't there before, we can skip that line.

Currently Scala Steward is not configured for the octopus. @krzemin I'll prepare a configuration for that if you don't mind. WDYT?

Reply:
That would be great @eltherion 👍

Reply:
That would be great @eltherion 👍

Reply:
Configured and merged: https://github.com/scala-steward-org/repos/pull/276
Since tomorrow Scala Steward should take care of updated deps in the project.

Reply:
Excellent, thank you 👍